orca-echo/src/main/groovy/com/netflix/spinnaker/orca/echo/pipeline/ManualJudgmentStage.groovy

```diff
@@ -75,18 +79,37 @@ class ManualJudgmentStage implements StageDefinitionBuilder, AuthenticatedStage
     @Autowired(required = false)
     EchoService echoService
 
+    @Autowired(required = false)
+    private FiatPermissionEvaluator fiatPermissionEvaluator
```

Comment:
While we're in here, I'd like to see these autowired properties be changed to a constructor instead:

```
private final EchoService echoService
private final FiatPermissionEvaluator fiatPermissionEvaluator

WaitForManualJudgmentTask(Optional<EchoService> echoService, Optional<FiatPermissionEvaluator> fpe) {
  this.echoService = echoService.orElse(null)
  this.fiatPermissionEvaluator = fpe.orElse(null)
}
```

Reply:
done